browser/android/brave_sync_worker.cc

```diff
@@ -346,7 +346,7 @@ std::string GetWordsFromSeedHex(const std::string& str_seed_hex) {
   std::string sync_code_words;
   if (base::HexStringToBytes(str_seed_hex, &bytes)) {
     DCHECK_EQ(bytes.size(), SEED_BYTES_COUNT);
-    if (bytes.size(), SEED_BYTES_COUNT) {
```

Comment:
this is definitely concerning, are we missing a test to cover this?

Reply:
cc: @AlexeyBarabash

Reply:
```
  std::array<uint8_t, SEED_BYTES_COUNT> bytes = {};
  if (base::HexStringToSpan(str_seed_hex, &bytes)) {
  }
```
to `if` only once